modules/raft/pom.xml

```diff
@@ -189,6 +158,20 @@
                         <version>${jmh.framework.version}</version>
                     </dependency>
                 </dependencies>
+                <configuration>
+                    <annotationProcessorPaths>
+                        <path>
+                            <groupId>org.apache.ignite</groupId>
+                            <artifactId>ignite-network-annotation-processor</artifactId>
+                            <version>${project.version}</version>
+                        </path>
+                        <path>
+                            <groupId>org.openjdk.jmh</groupId>
+                            <artifactId>jmh-generator-annprocess</artifactId>
```

Comment:
why is this change needed? The previous configuration was used to avoid the `jmh` processor during non-test build

Reply:
IDEA doesn't know about different executions, so it doesn't execute annotation processor at all for raft module. It's also inconsistent with other modules, which use jmh.